app/src/main/java/com/example/android_demo_kotlin/adapter/ExpandableRecyclerAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.android_demo_kotlin.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import android.widget.RelativeLayout
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.android_demo_kotlin.R
+import com.example.android_demo_kotlin.model.AppDemo
+
+class ExpandableRecyclerAdapter(val AppList: ArrayList<AppDemo>) : RecyclerView.Adapter<ExpandableRecyclerAdapter.ExpandableViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ExpandableViewHolder {
+        val view: View = LayoutInflater.from(parent.context).inflate(R.layout.item_recyclerview_expandable, parent, false)
+        return ExpandableViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: ExpandableViewHolder, position: Int) {
+        holder.apply {
+            nameTxt.text = AppList[position].namehello
```

Comment:
use `with` or `apply` for 
`nameTxt.text = AppList[position].namehello
dateTxt.text = AppList[position].date
apiTxt.text = AppList[position].apilevel
descriptionTxt.text = AppList[position].description`